app/src/androidTest/java/ch/epfl/sdp/blindly/PermissionTest.kt

```diff
@@ -26,7 +26,7 @@ class PermissionTest {
         init()
```

Comment:
I think that the test functions should all be renamed since the profile page names changed (since they were all of the form `ProfileNFiresProfileN`)

Reply:
Yeaah the thing is this task and issue went a bit further than expected after the meeting. It was at first just small changes on profile pages and error fixes for a previous PR, I will deal with these as well but the title of the task, estimates and actual time spent varied a lot.

Reply:
That's true. Thanks for doing it !